src/App.jsx

```diff
@@ -76,10 +68,14 @@ const App = () => {
 					path='/courses'
 					component={() => <Courses isLoading={isLoading} />}
```

Comment:
Fetch courses and loading should be inside Courses component
```suggestion
					component={Courses}
```

Reply:
fixed